vdc/scripts/users/user_data.py

```diff
@@ -0,0 +1,155 @@
+import shortuuid
+
+import google
+from google.cloud import storage
+from googleapiclient.errors import HttpError
+from globals import k8s, kube_client, gcloud_service
+from table import Table
+
+shortuuid.set_alphabet("0123456789abcdefghijkmnopqrstuvwxyz")
+
+
+def create_service_id():
+    return f'user-{shortuuid.uuid()}'
+
+
+def create_google_service_account(sa_name, google_project):
+    return gcloud_service.projects().serviceAccounts().create(
+            name=f'projects/{google_project}',
+            body={
+                "accountId": sa_name, "serviceAccount": {
+                    "displayName": "user"
```

Comment:
User is just a prefix to slightly help with selecting these ids (visually easier, and in some cases it may simplify queries on these selectors). To maximize id entropy, we can prefix using a small part of the user id, but I forewent this route because we can't really control the user id characterset.